marimo/_cli/sandbox.py

```diff
@@ -137,7 +160,12 @@ def construct_uv_flags(
     # NB. Used in quarto plugin
 
     # If name if a filepath, parse the dependencies from the file
-    dependencies = pyproject.requirements_txt_lines
+    try:
+        # Try exporting env with uv
+        dependencies = _uv_export_script_requirements_txt(pyproject.name)
```

Comment:
this could be a pyproject.toml (i think), so we could do a light check that it ends with `.py`

Reply:
When would that happen?

```sh
marimo edit . --sandbox
``` 

?

Reply:
yea exactly. you can start the sandbox without a script/file in mind. still ephemeral, but starts empty